app/Models/JobPoster.php

```diff
@@ -64,12 +65,37 @@
  * @method boolean isOpen()
  * @method string timeRemaining()
  */
-class JobPoster extends BaseModel {
+class JobPoster extends BaseModel
+{
 
     use \Dimsav\Translatable\Translatable;
     use Notifiable;
 
-    public $translatedAttributes = ['city', 'title', 'impact', 'branch', 'division', 'education'];
+    const DATE_FORMAT = [
```

Comment:
Date formatting should eventually be moved into a helper class (presumably we'll need to do this elsewhere) but it works for now.